### PR TITLE
track germs - work in process

### PR DESCRIPTION
**Pull request by LeahBineth**

### CODE REVIEW

File: 1 tables.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+create database TrackGermDB
+go
+use TrackGermDB
+go
+create table dbo.germ(
+	GermName varchar(200) not null,
```
Comment (mordechaigellis) [inline 1 tables.sql:7]: missing pk

File: 2 sample data.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+select 'Norovirus', 'human'
```
Comment (mordechaigellis) [inline 2 sample data.sql:2]: missing insert

**Review by mordechaigellis — CHANGES_REQUESTED** [review]
missing some basic sql

File: 1 tables.sql (hunk 1 above)
Comment (LeahBineth) [reply 1 tables.sql:7]: that was a mistake

File: 2 sample data.sql (hunk 2 above)
Comment (LeahBineth) [reply 2 sample data.sql:2]: will fix

File: 1 tables.sql (hunk 1 above)
Comment (LeahBineth) [reply 1 tables.sql:7]: fixed - https://github.com/mordechaigellis/track-germ/pull/5/commits/4666b47d634334ca8ec9c79c678fba4f4b91189f

File: 2 sample data.sql (hunk 2 above)
Comment (LeahBineth) [reply 2 sample data.sql:2]: fixed - https://github.com/mordechaigellis/track-germ/pull/5/commits/7c5045b88e7f519e4322ca2db469a86010cb8e49

File: 1 tables.sql (hunk 1 above)
Comment (mordechaigellis) [reply 1 tables.sql:7]: good

File: 2 sample data.sql (hunk 2 above)
Comment (mordechaigellis) [reply 2 sample data.sql:2]: all good

**Review by mordechaigellis — APPROVED** [review]

**Pull request closed without merging**